vmc/resource_vmc_site_recovery.go

```diff
@@ -72,6 +72,10 @@ func resourceSiteRecovery() *schema.Resource {
 
 func resourceSiteRecoveryCreate(d *schema.ResourceData, m interface{}) error {
 
+	err := (m.(*ConnectorWrapper)).authenticate()
```

Comment:
does this check if the current token is valid or not and if not, get a new token OR it always gets a new token?

Reply:
authenticate() creates a new access token with a validity of 30 minutes. There is no other API but the POST API /am/api/auth/api-tokens/authorize to refresh the access token currently under this authentication mechanism. The access token can be refreshed only in the OAuth authentication mechanism using the refresh_token grant type.

Reply:
I think we should rename this func https://github.com/terraform-providers/terraform-provider-vmc/blob/master/vmc/provider.go#L27. Quite misleading.

Reply:
Yes I agree.